app/proto/reporting.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package org.couchers.api.reporting;
+
+import "google/protobuf/empty.proto";
+
+import "annotations.proto";
+
+service Reporting {
+  option (auth_level) = AUTH_LEVEL_SECURE;
+
+  rpc Report(ReportReq) returns (google.protobuf.Empty) {
+    // Report content, user, or something else
+  }
+}
+
+message ReportReq {
+  // e.g. spam, inappropriate
+  string reason = 1;
+  // short description
+  string description = 2;
+  // a reference to the content being reported, this should be filled out by the UI automatically
+  string content_ref = 3;
+  // the id or username of the author of this piece of content
+  string author_user = 4;
+
+  string user_agent = 5;
+  // url that the user was on when they reported
+  string page = 6;
```

Comment:
Rename to url to not confuse with what we call "pages" in the API. Also what should non-web clients send here?

Reply:
It's consistent with bug reporting, so I'd prefer to keep as-is